### PR TITLE
routes-and-pages

### PR DESCRIPTION
**Pull request by Sharmaryan**
**added basic routes, and pages**

Screenshot (62)
Screenshot (63)
Screenshot (64)
Screenshot (65)
Screenshot (66)

### CODE REVIEW

File: src/components/Shop/Shop.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const Shop = () => {
+  return (
+    <div>
+      <main className="container">
+        <h1 className="container-title">store for bibliophiles</h1>
+
+        <button class="btn btn-link">
+          <a href="#" class="btn-text-decoration btn-shop ">
+            shop now
+          </a>
```
Comment (romabulani) [inline src/components/Shop/Shop.js:12]: Suggestion : You can use the useNavigate hook here instead of using anchor tag

**Review by romabulani — COMMENTED** [review]
Suggstion : src/components/Navbar/Navbar.jsx is an empty file, you can delete i if not required

**Review by romabulani — APPROVED** [review]
Nice folder structure and clean code, have suggested few changes if you have bandwidth to complete them.

**Comment by Sharmaryan** [comment]
> Suggstion : src/components/Navbar/Navbar.jsx is an empty file, you can delete i if not required

it was css file and it was empty , I deleted it

File: src/components/Shop/Shop.js (hunk 1 above)
Comment (Sharmaryan) [reply src/components/Shop/Shop.js:12]: sure, I will use it

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Navbar />
+      <Routes>
```
Comment (bharati-21) [inline src/App.js:9]: You can create a separate component for routes and them import it here. This way App Component looks clean.

**Review by bharati-21 — APPROVED** [review]
Just dropped a suggestion you could look into.

Apart from that all looks great, you can go ahead and merge.

File: src/App.js (hunk 2 above)
Comment (Sharmaryan) [reply src/App.js:9]: sure, thanks for suggestion

Comment (Sharmaryan) [reply src/App.js:9]: done with the changes

**Pull request merged**